dist-unminified/recorder.js

```diff
@@ -197,7 +201,9 @@ Recorder.prototype.initSourceNode = function( sourceNode ){
   var self = this;
   return global.navigator.mediaDevices.getUserMedia({ audio : this.config.mediaTrackConstraints }).then( function( stream ){
     self.stream = stream;
-    return self.audioContext.createMediaStreamSource( stream );
+    self.microphone = self.audioContext.createMediaStreamSource( stream );
+    self.microphone.connect( self.volumeNode );
```

Comment:
I don't agree that we put this volume node before the monitor gain node.  Doing double gain on the monitor stream.  I think it would be better if they change their outputs independently.